ocp_resources/reclaim_space_job.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+from typing import Any, Optional
+from ocp_resources.resource import NamespacedResource, MissingRequiredArgumentError
+
+
+class ReclaimSpaceJob(NamespacedResource):
+    """
+    https://github.com/csi-addons/kubernetes-csi-addons/blob/main/apis/csiaddons/v1alpha1/reclaimspacejob_types.go
+    """
+
+    api_group = NamespacedResource.ApiGroup.CSIADDONS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        backoff_limit: Optional[int] = None,
+        target: dict[str, Any] | None = None,
+        retry_deadline_seconds: Optional[int] = None,
+        timeout_seconds_reclaim_job: int | None = None,
+        **kwargs: Any,
+    ) -> None:
+        """
+         Args:
+        backoff_limit (int, Optional): The number of retries for a reclaim space job.
+        target (dict): Volume target on which the operation would be performed.
+        retryDeadlineSeconds (int, Optional): Duration in seconds relative to the start time that the
+            operation may be retried.
+        timeout_seconds_reclaim_job (int, Optional): Specifies the timeout in seconds for the grpc request sent to
+            the CSI driver. If set, minimum allowed value is 60
```

Comment:
please remove `If set, minimum allowed value is 60` - this is an API config and may change; this is not set in the wrapper's code